src/Shared/EnumParserHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Avalonia
+{
+    internal class EnumParserHelper
```

Comment:
Just add it to the Avalonia.Base.
It has internal helpers and it has internal access attribute. Shared files are a bit a pain.

Reply:
> Just add it to the Avalonia.Base.

The helper is also referenced in `Avalonia.DesignerSupport`, if we do this we have to make it public.

Reply:
> it has internal access attribute.

Oops didn't notice it before. I will change it now.

Reply:
You can add internal visibility attribute here https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Base/Properties/AssemblyInfo.cs#L31
We manage internal helpers in this way usually
See https://github.com/AvaloniaUI/Avalonia/tree/master/src/Avalonia.Base/Utilities

Reply:
Done.

Reply:
I didn't do the same for `src\Shared\SourceGeneratorAttributes.cs`, otherwise the source generator will have to reference `Avalonia.Base`.